executor/src/witgen/eval_result.rs

```diff
@@ -168,6 +168,12 @@ pub enum EvalError<T: FieldElement> {
     Multiple(Vec<EvalError<T>>),
 }
 
+impl<T: FieldElement> Debug for EvalError<T> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        <Self as fmt::Display>::fmt(self, f)
+    }
+}
+
```

Comment:
This way, when you do `.unwrap()` on the error, it prints the rendered string, not the debug version of it (with `"\n"` instead of actual new lines).